lms/models/assignment.py

```diff
@@ -87,7 +87,7 @@ class Assignment(CreatedUpdatedMixin, Base):
         "AssignmentMembership", lazy="dynamic", viewonly=True
     )
 
-    course_id: Mapped[int | None] = mapped_column(sa.ForeignKey(Course.id))
+    course_id: Mapped[int | None] = mapped_column(sa.ForeignKey(Course.id), index=True)
```

Comment:
I don't see a migration for this change.

Reply:
Good catch!

Not sure what happen here, I use alembic autogenerate to get the first draft of these :thinking: 

I'm going to get in the habit of running the following in the migrations testing notes:

`tox -e dev --run-command 'alembic check'`

which fails if alembics dects model changes that are not sync with the DB

Let's discuss how we could integrate that into CI or similar flows.